### DISCUSSION TITLE
Zurücksetzen der Ladestatistik

### DISCUSSION THREAD

**Discussion by HellClown2k** [pkuehnel/TeslaSolarCharger#1024] [Q&A]
Hallo, 
gibt es irgend eine möglichkeit die Statistik zum Jahresende einmal auf 0 zu setzen?
Würde damit gerne für das nächste einen besseren Überblick bekommen. 

Danke im voraus

![EADFA04F-7146-430A-8F9C-F035DC876926_4_5005_c](https://github.com/pkuehnel/TeslaSolarCharger/assets/102475348/2c948b80-57a2-42ac-96dc-e3b121ebac98)

**Comment by pkuehnel** [comment]
Aktuell leider nicht.
